### PR TITLE
added titles to the docs pages

### PR DESCRIPTION
**Pull request by tusharojha**
- added title properties of docs pages for better seo
- changed bounties page -> link to subsocial bounties repository

### CODE REVIEW

File: docs/develop/sdk/follow/CreateFollow.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ id: create-follow
 title: Following And Unfollowing
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Create Following on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/follow/CreateFollow.md:8]:
```suggestion
  <title>Following And Unfollowing With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/follow/FetchFollow.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ id: fetch-follow
 title: Fetch Follows
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Followers on Subsocial | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/follow/FetchFollow.md:8]:
```suggestion
  <title>How To Get Followers With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/posts/FetchPosts.md [hunk 3]
```diff
@@ -3,6 +3,11 @@ id: fetch-posts
 title: Fetch Posts
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Posts on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/posts/FetchPosts.md:8]:
```suggestion
  <title>How To Get Posts With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/profiles/FetchProfiles.md [hunk 4]
```diff
@@ -3,6 +3,11 @@ id: fetch-profiles
 title: Fetch Profiles
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Profiles on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/profiles/FetchProfiles.md:8]:
```suggestion
  <title>How To Profiles With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/reactions/FetchReactions.md [hunk 5]
```diff
@@ -3,6 +3,12 @@ id: fetch-reactions
 title: Fetch Reactions
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Reactions on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/reactions/FetchReactions.md:8]:
```suggestion
  <title>How To Get Reactions With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/spaces/FetchSpaces.md [hunk 6]
```diff
@@ -3,6 +3,11 @@ id: fetch-spaces
 title: Fetch Spaces
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Subsocial Spaces | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/spaces/FetchSpaces.md:8]:
```suggestion
  <title>How To Get Subsocial Spaces With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/usernames/FetchUsername.md [hunk 7]
```diff
@@ -3,6 +3,11 @@ id: fetch-usernames
 title: Fetching Usernames
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Fetch Subsocial Usernames | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/usernames/FetchUsername.md:8]:
```suggestion
  <title>How To Get Subsocial Usernames With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/posts/CreatePosts.md [hunk 8]
```diff
@@ -3,6 +3,11 @@ id: create-posts
 title: Create and update Posts
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Create and Update Posts on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/posts/CreatePosts.md:8]:
```suggestion
  <title>Creating and Updating Posts With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/posts/PostWithDetails.md [hunk 9]
```diff
@@ -4,6 +4,10 @@ title: Posts With Details
 displayed_sidebar: developSidebar
 ---
 
+<head>
+  <title>Alternate Methods to Fetch Posts on Subsocial | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/posts/PostWithDetails.md:8]:
```suggestion
  <title>Alternate Methods To Get Posts With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/profiles/CreateProfiles.md [hunk 10]
```diff
@@ -3,6 +3,11 @@ id: create-profiles
 title: Create And Update Profiles
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Create and Update Profile on the Subsocial Blockchain | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/profiles/CreateProfiles.md:8]:
```suggestion
  <title>Creating and Updating Profiles With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/reactions/CreateReactions.md [hunk 11]
```diff
@@ -3,6 +3,11 @@ id: create-reactions
 title: Create, Update, And Delete Reactions
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Create Reactions on Subsocial | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/reactions/CreateReactions.md:8]:
```suggestion
  <title>How To Create Reactions With The Subsocial JS SDK</title>
```

File: docs/develop/sdk/spaces/CreateSpaces.md [hunk 12]
```diff
@@ -3,6 +3,11 @@ id: create-spaces
 title: Create And Update Spaces
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Create & Update Subsocial Spaces | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/spaces/CreateSpaces.md:8]:
```suggestion
  <title>How To Create & Update Subsocial Spaces With The JS SDK</title>
```

File: docs/develop/sdk/usernames/RegisterUsername.md [hunk 13]
```diff
@@ -3,6 +3,11 @@ id: register-usernames
 title: Registering Usernames
 displayed_sidebar: developSidebar
 ---
+
+<head>
+  <title>Register Subsocial Usernames | JS SDK Guide</title>
```
Comment (Yung-Beef) [inline docs/develop/sdk/usernames/RegisterUsername.md:8]:
```suggestion
  <title>Registering Subsocial Usernames With The Subsocial JS SDK</title>
```

File: docs/develop/zero-to-hero/twitter-dapp/ProjectSetup.md [hunk 14]
```diff
@@ -4,6 +4,10 @@ title: Introduction
 displayed_sidebar: developSidebar
 ---
 
+<head>
+  <title>Build Decentralised Twitter in 50 minutes using Subsocial | Zero-to-Hero Tutorial</title>
```
Comment (Yung-Beef) [inline docs/develop/zero-to-hero/twitter-dapp/ProjectSetup.md:8]:
```suggestion
  <title>Build A Decentralized Twitter In 50 Minutes With Subsocial</title>
```

File: docs/develop/zero-to-hero/twitter-dapp/ProjectSetup.md [hunk 15]
```diff
@@ -4,6 +4,10 @@ title: Introduction
...
+</head>
+
 The **Zero to Hero tutorial** is your chance to build a fully decentralized twitter app in 50 minutes using the Subsocial Starter.
```
Comment (Yung-Beef) [inline docs/develop/zero-to-hero/twitter-dapp/ProjectSetup.md:11]:
```suggestion
The **Zero to Hero tutorial** is your chance to build a fully decentralized Twitter app in 50 minutes using the Subsocial Starter.
```

**Review by Yung-Beef — APPROVED** [review]

File: docs/develop/sdk/APITypes.md [hunk 16]
```diff
@@ -6,7 +6,7 @@ displayed_sidebar: developSidebar
 
 ## General API
```
[the remark this answers is not in the document]
Comment (tusharojha) [reply docs/develop/sdk/APITypes.md:8]: Understanding the API Types of Subsocial JS SDK

**Pull request merged**